secretmanager/snippets/regional_samples/delete_regional_secret_label.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python
+
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+
+# [START secretmanager_delete_regional_secret_label]
+import argparse
+
+# Import the Secret Manager client library.
+from google.cloud import secretmanager_v1
+
+
+def delete_regional_secret_label(
+    project_id: str, location_id: str, secret_id: str, label_key: str
+) -> secretmanager_v1.UpdateSecretRequest:
+    """
+    Delete a label on an existing secret.
+    """
+
+    # Endpoint to call the regional secret manager sever.
```

Comment:
Instead of `sever`, you can us `Google API` or `secret manager API`.

Reply:
Fixed